omod/src/main/webapp/locationMapping.jsp

```diff
@@ -35,7 +37,8 @@
                 </c:forEach>
                 <input name="savedOrgUnitUuidOf_${location.uuid}" type="hidden" value="${savedOrgUnitUuid}">
                 <td>
-                    <select name="orgUnitOf_${location.uuid}">
+                    <select name="orgUnitOf_${location.uuid}"
+                            <openmrs:hasPrivilege privilege="Manage Location Mappings" inverse="true">disabled</openmrs:hasPrivilege>>
```

Comment:
It seems the user cannot access this page if the user doesn't have the `Manage Location Mappings` permission. So this is an unreachable code I guess.

Reply:
I went through it again. Ignore this comment.